pkg/didcomm/protocol/introduce/service.go

```diff
@@ -0,0 +1,22 @@
+/*
+Copyright SecureKey Technologies Inc. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package introduce
+
+import "github.com/hyperledger/aries-framework-go/pkg/common/metadata"
+
+const (
+	// Introduce protocol name
+	Introduce = "introduce"
+	// Spec defines the introduce spec
+	Spec = metadata.AriesCommunityDID + ";spec/introduce/1.0/"
+	// ConnectionProposal defines the introduce proposal message type.
+	ConnectionProposal = Spec + "proposal"
```

Comment:
May be we can use "Introduce" prefix here as this is an introduce protocol.

Reply:
I did that on purpose, my IDE (GoLand) says `Name starts with package name`

Reply:
@soluchok I don't understand your comment? This is package `introduce`. Also, we're missing the [`request`](https://github.com/hyperledger/aries-rfcs/tree/master/features/0028-introduce#request) message.